packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -151,6 +151,7 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   const inputRef = React.useRef<HTMLInputElement>(null);
   const menuRef = React.useRef<HTMLUListElement>(null);
   const toggleButtonRef = React.useRef<HTMLSpanElement>(null);
+  const firstRenderRef = React.useRef(true);
```

Comment:
Maybe invert and rename this to something like `justMounted` or `mounted`.

Reply:
Why? What is the difference?

Reply:
Yeah, I think we used flag like that in somewhere else already.
@gretanausedaite `mounted` came from the original react in classes.
https://reactjs.org/docs/react-component.html#componentdidmount
so `useEffect` first call is 'almost' same as that lifecycle method in class components.